CHANGELOG.md

```diff
@@ -26,6 +26,7 @@
 - `coralogixexporter` Allow exporter timeout to be configured (#7957)
 - `prometheusremotewriteexporter` support adding trace id and span id attached to exemplars (#8380)
 - `influxdbexporter`: accept histogram metric missing infinity bucket. (#8462)
+- `prometheusremotewriteexporter`: Translate resource attributes to the target info metric (#8493)
```

Comment:
Make sure this goes in the `Unreleased` section.

Reply:
done